core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -1619,15 +1626,15 @@ public String toString() {
           .build();
   public static final PropertyKey USER_RPC_RETRY_MAX_NUM_RETRY =
       new Builder(Name.USER_RPC_RETRY_MAX_NUM_RETRY)
-          .setDefaultValue(20)
+          .setDefaultValue(100)
           .setDescription("Alluxio client RPCs automatically retry for transient errors with "
               + "an exponential backoff. This property determines the maximum number of "
               + "retries.")
           .build();
   public static final PropertyKey USER_RPC_RETRY_MAX_SLEEP_MS =
```

Comment:
When Configuration is initialized, we could give a warning if this property is less than `MASTER_SAFEMODE_WAIT`

Reply:
Good idea.